tox.ini

```diff
@@ -391,34 +392,36 @@ deps =
     -r{toxinidir}/requirements.txt
     func-s_{base,metaflow}-py{36,37,38,39,310}: -r{toxinidir}/requirements_dev.txt
     pytest-mock
-    yea-wandb=={env:YEA_WANDB_VERSION}
+    https://github.com/wandb/yea/archive/yea-0.8.zip
+    https://github.com/wandb/yea-wandb/archive/yea-wandb-0.8.zip
+    ; yea-wandb=={env:YEA_WANDB_VERSION}
```

Comment:
make sure to fix this!

Reply:
yea-wandb and yea were released so i have updated